chart/templates/role.yaml

```diff
@@ -74,10 +74,13 @@ rules:
     resources: ["networkpolicies"]
     verbs: ["create", "delete", "patch", "update", "get", "list", "watch"]
   {{- end }}
+  - apiGroups: [ "snapshot.storage.k8s.io" ]
+    resources: [ "volumesnapshots" ]
+    verbs: ["create", "delete", "patch", "update", "get", "list"]
   {{- if .Values.sync.toHost.volumeSnapshots.enabled }}
   - apiGroups: ["snapshot.storage.k8s.io"]
     resources: ["volumesnapshots"]
-    verbs: ["create", "delete", "patch", "update", "get", "list", "watch"]
+    verbs: ["watch"]
```

Comment:
Are you sure you don't need the watch here?

Reply:
Yup. vCluster is not watching VolumeSnapshots, it's the upstream snapshot-controller that is watching them.

We could implement watching VolumeSnapshots, as an optimization, so that vCluster reacts directly to VolumeSnapshot changes, as opposed to periodically checking them. If we want to do this, I would rather make that change in a separate PR, as this is already big enough and we want QA to start testing as soon as possible.